patches/third_party-polymer-v3_0-BUILD.gn.patch

```diff
@@ -1,24 +1,35 @@
 diff --git a/third_party/polymer/v3_0/BUILD.gn b/third_party/polymer/v3_0/BUILD.gn
-index f1166be2b458d7796dcea5504a35f1099eb3190d..f75e4665a667e6a515d0bc33b6f5f7b47e2df30b 100644
+index 75fe84796f2bce02f7afff54e6e335a4ba784fe6..5969acf99c416ab2646b1573c58bc1b6a98dee0a 100644
 --- a/third_party/polymer/v3_0/BUILD.gn
 +++ b/third_party/polymer/v3_0/BUILD.gn
-@@ -43,6 +43,7 @@ if (is_chromeos) {
+@@ -39,10 +39,12 @@ js_files = [
+ # Also needed on LaCros for Print Preview. Remove once CrOS Print Preview
+ # launches.
+ if (is_chromeos) {
++} if (!is_android) {
```

Comment:
why do we need these patches in the first place?

Reply:
We are using iron-dropdown in our Settings WebUI. At some point upstream limited it to chromeos because they weren't using it anywhere else any more.

Reply:
can't we just include it ourselves instead of patching here? This seems like quite a lot of patches just to include some files.

Reply:
I think if we wanted to include it we'd have to get a number of files. We only use it one place on brave://settings/wallet/networks when you want to `Add` a network. Perhaps the wallet team can use something else, like a regular dropdown? cc: @brave/crypto-wallets-front-end

Reply:
we're doing this for one place in wallet settings? I don't think all of these patches makes sense for that and we should follow-up to either include this some other way (cc @petemill) or just change the dropdown.

Reply:
I'm on PTO so can't look in to it too much but another option is to use Brave's Nala Dropdown which can be used directly.

Reply:
Although the styling is different and migration would be a tiny bit involved.

Reply:
can we create a follow-up for this?

Reply:
Created https://github.com/brave/brave-browser/issues/40292